alerting/src/main/kotlin/org/opensearch/alerting/workflow/CompositeWorkflowRunner.kt

```diff
@@ -1,30 +1,90 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
 package org.opensearch.alerting.workflow
 
+import org.opensearch.alerting.BucketLevelMonitorRunner
+import org.opensearch.alerting.DocumentLevelMonitorRunner
 import org.opensearch.alerting.MonitorRunnerExecutionContext
+import org.opensearch.alerting.QueryLevelMonitorRunner
 import org.opensearch.alerting.model.MonitorRunResult
+import org.opensearch.alerting.util.isDocLevelMonitor
+import org.opensearch.commons.alerting.model.CompositeInput
 import org.opensearch.commons.alerting.model.Workflow
+import org.opensearch.commons.alerting.util.isBucketLevelMonitor
 import java.time.Instant
+import java.util.UUID
+
+object CompositeWorkflowRunner : WorkflowRunner() {
 
-class CompositeWorkflowRunner : WorkflowRunner() {
     override suspend fun runWorkflow(
         workflow: Workflow,
         monitorCtx: MonitorRunnerExecutionContext,
         periodStart: Instant,
         periodEnd: Instant,
         dryRun: Boolean
-    ): MonitorRunResult<*> {
-        TODO("Not yet implemented")
-    }
+    ): List<MonitorRunResult<*>> {
+        val workflowExecutionId = UUID.randomUUID().toString()
+
+        val delegates = (workflow.inputs[0] as CompositeInput).sequence.delegates.sortedBy { it.order }
+        // Fetch monitors by ids
+        val monitors = monitorCtx.workflowService!!.searchMonitors(delegates.map { it.monitorId }, delegates.size, workflow.owner)
+
+        // Validate the monitors size
+        if (delegates.size != monitors.size) {
+            val diffMonitorIds = delegates.map { it.monitorId }.minus(monitors.map { it.id }.toSet()).joinToString()
+            throw IllegalStateException("Delegate monitors don't exist $diffMonitorIds")
```

Comment:
Plz also log workflow id in the message

Reply:
Will do. Also will add a logs on the beginning and end of workflow execution